src/components/Menu/Menu.tsx

```diff
@@ -119,11 +137,14 @@ class Menu extends AutoControlledComponent<Extendable<MenuProps>, any> {
       secondary,
       underlined,
       vertical,
+      submenu,
     } = this.props
     const { activeIndex } = this.state
 
-    return _.map(items, (item, index) =>
-      MenuItem.create(item, {
+    return _.map(items, (item, index) => {
+      const active =
+        typeof activeIndex === 'string' ? parseInt(activeIndex, 10) : activeIndex === index
```

Comment:
This condition is not valid, if we `activeIndex` is `string` it will be converted to `number`, in other cases it will be boolean:

```js
let activeIndex = 1
let index = 1
let active

active = typeof activeIndex === 'string' ? parseInt(activeIndex, 10) : activeIndex === index
console.log(typeof active) // boolean

activeIndex = '1'
active = typeof activeIndex === 'string' ? parseInt(activeIndex, 10) : activeIndex === index
console.log(typeof active) // number
```

Reply:
Yep, I am missing brackets
` typeof activeIndex === 'string' ? parseInt(activeIndex, 10) : activeIndex === index` => ` (typeof activeIndex === 'string' ? parseInt(activeIndex, 10) : activeIndex) === index
`

Reply:
The active prop should always be boolean. Fixed.